decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -63,6 +67,9 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
                                      y: self.view.frame.origin.y,
                                      width: self.view.frame.width,
                                      height: window.origin.y + window.height - keyboardSize.height)
+            
+            // adjust focus seed word input just above the keyboard
```

Comment:
This does not adjust the focused seed word input or any input for that matter. It creates space between the bottom of the table and the top of the keyboard. A better comment would be:
`// add space between bottom of table and top of keyboard`

Reply:
the space is meant to adjust the view